akka-stream/src/main/scala/akka/stream/impl/FanoutProcessor.scala

```diff
@@ -6,7 +6,7 @@ package akka.stream.impl
 
 import akka.actor.{ Actor, ActorRef, Deploy, Props }
 import akka.annotation.{ DoNotInherit, InternalApi }
-import akka.stream.{ ActorMaterializerSettings, Attributes }
+import akka.stream.{ ActorMaterializerSettings, Attributes, StreamSubscriptionTimeoutTerminationMode }
```

Comment:
Aren't we putting these all on their own lines now, wonder if scalafmt or IntelliJ should have caught that.

Reply:
It's a manual thing, scalafmt can't detect afair.